deployment/v3/utils/readuser-util/install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Installs readuser-util
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=readuser-util
+IAM_NS=keycloak
+CHART_VERSION=0.0.1-develop
+
+echo Create $NS namespace
+kubectl create ns $NS
+
+function installing_readuser-util() {
+
+  echo Installing readuser-util
+  helm -n $NS install readuser-util mosip/readuser-utill --version $CHART_VERSION -f values.yaml --wait
+  helm -n $IAM_NS install readuser-iam-init mosip/keycloak-init --version $CHART_VERSION -f readuser-init-values.yaml --wait
```

Comment:
run keycloak init for readuser from same readuser-util namespace.

Reply:
Updated